### PR TITLE
Tao 1937 proctor form

### PR DESCRIPTION
**Pull request by ssipasseuth**
note 1 : there is currently no unit test for the form component because it rely on server form generation + validation.
note 2 :  the form submission can create a proctor but it will fails if the test site admin is not assigned to any test center. This situation should not happen since the ui should prevent this eventually.

### CODE REVIEW

File: views/js/component/proctorForm.js [hunk 1]
```diff
@@ -24,54 +24,117 @@ define([
...
+        users.checkLogin(formData.loginId, proctorLoginCheckUrl);
     }
-    
+
+    /**
+     * Create the form in the container
+     * @param {type} $container
+     * @returns {undefined}
+     */
     function init($container){
```
Comment (jsconan) [inline views/js/component/proctorForm.js]: I am a little bit confused... Is this function still in use ? The factory below makes the same thing...

File: views/js/component/proctorForm.js [hunk 2]
```diff
@@ -24,54 +24,117 @@ define([
...
+                            testCenters : [] //@todo get the testCenter from the test center datalist component (config.testCenterList)
+                        };
+                        _.each(fields, function(field){
+                            data[field.name] = field.value;
+                        });
+
+                        $.post(proctorFormUrl, data, function(res){
+                            if(res.created){
+                                feedback().success(__('Proctor created'));
+                                $element.empty();
```
Comment (jsconan) [inline views/js/component/proctorForm.js]: I think there is something missing: we need to known when the component has done its stuff, in order to go back to the list. Maybe an event or a callback ?

File: controller/ProctorManager.php [hunk 3]
```diff
@@ -180,23 +181,10 @@ public function createProctorForm(){
         
         return $this->returnJson(array(
```
Comment (jsconan) [inline controller/ProctorManager.php]: The `returnJson` method does not return a value, so it is useless to apply a `return` here.

File: views/js/component/proctorForm.js (hunk 2 above)
Comment (ssipasseuth) [inline views/js/component/proctorForm.js]: Yes, let's figure it out during the integration phase.

**Comment by ssipasseuth** [comment]
Thabks for your feedback, Ready for a new review !

**Pull request merged**